deployment/network-operator/values.yaml

```diff
@@ -179,7 +179,8 @@ ofedDriver:
       enable: true
```

Comment:
should we add upgrade policy chart parameters to `deployment/README` ?

Reply:
Let's do it in a separate PR